### ISSUE TITLE
plot.coastline(..., add=TRUE) has "fill" error

### ISSUE THREAD

**Issue by richardsc** [dankelley/oce#884]
Must be a remnant from the recent argument renaming:

```r
library(oce)
data(coastlineWorld)
plot(coastlineWorld)
plot(coastlineWorld, add=TRUE)
```
```
Error in polygon(longitude, latitude, border = border, col = fill, ...) : 
  argument "fill" is missing, with no default
```

**Comment by richardsc** [comment]
Easy fix in PR#885. I'll close after you merge.

**Comment by richardsc** [comment]
Closing
